pbm/archive/conv.go

```diff
@@ -0,0 +1,133 @@
+package archive
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"runtime"
+	"slices"
+	"strings"
+
+	mtArchive "github.com/mongodb/mongo-tools/common/archive"
+	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
+
+	"github.com/percona/percona-backup-mongodb/pbm/storage"
+)
+
+func GenerateV1FromV2(stg storage.Storage, bcp, rs string) error {
+	metaV2Filename := fmt.Sprintf("%s/%s/%s", bcp, rs, MetaFileV2)
+	rdr, err := stg.SourceReader(metaV2Filename)
+	if err != nil {
+		return errors.Wrapf(err, "open file: %q", metaV2Filename)
+	}
+	defer rdr.Close()
+
+	data, err := io.ReadAll(rdr)
+	if err != nil {
+		return errors.Wrap(err, "read")
+	}
+	rdr.Close()
```

Comment:
```suggestion
	err = rdr.Close()
	if err != nil {
		return errors.Wrap(err, "close")
	}
```
It'll be cleaner to add error handling here.